presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -78,10 +82,19 @@ protected QueryRunner createQueryRunner()
         return PrestoSparkQueryRunner.createHivePrestoSparkQueryRunner(configs);
     }
 
+    @Override
+    protected ExpectedQueryRunner createExpectedQueryRunner()
+            throws Exception
+    {
+        LocalQueryRunner runner = new LocalQueryRunner(TEST_SESSION);
+        runner.createCatalog(TEST_SESSION.getCatalog().get(), new TpchConnectorFactory(), ImmutableMap.of());
+        return runner;
+    }
+
     @Test
     public void testNativeExecutionWithProjection()
     {
-        assertUpdate("CREATE TABLE test_order WITH (format = 'DWRF') as SELECT orderkey, custkey FROM orders LIMIT 100", 100);
+        assertUpdate("CREATE TABLE test_order WITH (format = 'DWRF') as SELECT orderkey, custkey FROM orders WHERE orderkey <= 200", 55);
```

Comment:
Let's find a way to avoid hard-coding results, even result counts. Let's create a helper function that tables just SELECT ... query and does the following:

1- Run CREATE TABLE xxx AS <select query> and capture the number of rows.
2- Run <select query> using "expected" query runner and capture the number of rows. Assert number of rows from 1 and 2 match.
3- Verify that results of SELECT * FROM xxx and match results of <select query> using "expected" query runner.

To make this a bit more efficient, (2) may capture full result and then assert that number of rows in (1) is correct and that SELECT * FROM xxx returns the same result.